pkg/mutation/stats_reporter_test.go

```diff
@@ -0,0 +1,85 @@
+package mutation
+
+import (
+	"testing"
+
+	"go.opencensus.io/stats/view"
+)
+
+func TestReportIterationConvergence(t *testing.T) {
+	r, err := NewStatsReporter()
+	if err != nil {
+		t.Errorf("newStatsReporter() error %v", err)
+	}
+
+	const (
+		successMax = 5
+		successMin = 3
+		failureMax = 8
+		failureMin = failureMax
+	)
+
+	err = r.ReportIterationConvergence(SystemConvergenceTrue, successMax)
+	if err != nil {
+		t.Errorf("reportIterationConvergence error: %v", err)
```

Comment:
capitalize to match method name. If a linter complains that errors shouldn't start with a capital letter, just prefix with "got ". For example, "got ReportIterationConvergence error: %v"

Reply:
done